insights/parsers/insights_client_tags.py

```diff
@@ -0,0 +1,52 @@
+"""
+InsightsClientTags - file ``/etc/insights-client/tags.yaml``
+============================================================
+"""
+
+from insights import parser
+from insights.core import YAMLParser
+from insights.specs import Specs
+
+
+@parser(Specs.insights_client_tags)
+class InsightsClientTags(YAMLParser):
+    """
+    This class parses the YAML-formatted file `/etc/insights-client/tags.yaml`.
+    The file is created by the user after running this command:
+
+    `insights-client --group=<group-name>`
+
+    The command above creates the file and adds an initial group tag
+    as defined by the `--group` option.
+    Other tags are added manually by the user.
+
+    Sample input data::
+
+        group: eastern-sap
+        location: Boston
+        description:
+        - RHEL8
+        - SAP
+        key 4: value
+
+    Examples:
+        >>> type(tags)
+        <class 'insights.parsers.insights_client_tags.InsightsClientTags'>
+        >>> tags.data
+        {'group': 'eastern-sap', 'location': 'Boston', 'description': ['RHEL8', 'SAP'], 'key 4': 'value'}
+        >>> tags.group
+        'eastern-sap'
+        >>> tags.get('location')
+        'Boston'
+        >>> tags.get('key 1', 'not defined')
+        'not defined'
+    """
+
+    @property
+    def group(self):
```

Comment:
Is it necessary to have a `@property` for this case? The attribute can be set directly in `__init__`.

Moreover, it seems this only duplicates already existing value.
See `assert tags.group == 'eastern-sap'` and `assert tags.get("group") == 'eastern-sap'` in tests.